src/component/box-resizable/lib/box-resizable.ts

```diff
@@ -102,6 +102,7 @@ export class BoxResizable extends HTMLElement {
         this.previousSize = offsetWidth;
 
         resizable.style.width = `${width}px`;
+        window.dispatchEvent(new Event("resize"));
```

Comment:
Todo: listen to parent events instead https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserverEntry/contentBoxSize